use std::{
    error::Error,
};

fn main() -> Result<(), Box<dyn Error>> {
    ow_time::run()
}

